src/microbe_stage/editor/CellEditorComponent.GUI.cs

```diff
@@ -2,7 +2,10 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
+using AngleSharp.Common;
```

Comment:
What's used from AngleSharp here? AngleSharp is a HTML parsing library so I'd really prefer to keep it entirely outside gameplay code.